packages/reakit-system/src/__tests__/useCreateElement-test.tsx

```diff
@@ -31,6 +32,21 @@ test("render props", () => {
   `);
 });
 
+test("render props with component type", () => {
```

Comment:
This test case, at least to me, is very confusing. Do you think we could add an attribute to the div rendered by `A`? I think that would clear up which div is which!

Reply:
Yeah! You're right! I think the first div is the one that react-testing-library adds automatically. I'll update it.

Reply:
Ah yeah, I forgot about the container div. Feel free to resolve this thread once you're done 😄